src/custom/pages/Claim/InvestmentFlow.tsx

```diff
@@ -0,0 +1,216 @@
+import {
```

Comment:
Moves investment html code from index into this file, adds global claim state null checks and approveState / approveCallback from `claim` into this file as props

no internal changes